actors/builtin/verifreg/verified_registry_test.go

```diff
@@ -825,7 +807,19 @@ func (h *verifRegActorTestHarness) generateAndAddVerifierAndVerifiedClient(rt *m
 	h.addVerifiedClient(rt, verifier.Address, client.Address, client.Allowance)
 }
 
-func (h *verifRegActorTestHarness) addVerifiedClient(rt *mock.Runtime, verifier, client address.Address, allowance verifreg.DataCap) {
+func (h *verifRegActorTestHarness) addVerifiedClient(rt *mock.Runtime, verifier, client address.Address, caps ...verifreg.DataCap) {
+	// caps[0] expects allowance to be set
+	// caps[1] (optional) expected allowance to be verified
+	// additional args are ignored
```

Comment:
Fail the test rather than ignore an unexpected argument. Fail also if the slice has length zero.

Alternatively, I would be just as happy if the two args were explicit, and the usually-redundant expected cap was always explicitly passed.

Reply:
I want to change this to two explict arguments.